#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_link.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP:

@DPATCH@

--- x264-0.svn20070613/Makefile	2007-06-13 15:52:11.000000000 +0200
+++ x264-0.svn20070613/Makefile	2007-06-13 15:52:41.000000000 +0200
@@ -79,7 +79,8 @@
 $(SONAME): .depend $(OBJS) $(OBJASM)
 	$(CC) -shared -o $@ $(OBJS) $(OBJASM) -Wl,-soname,$(SONAME) $(LDFLAGS)
 
-x264$(EXE): $(OBJCLI) libx264.a 
+x264$(EXE): $(SONAME)
+x264$(EXE): $(OBJCLI)
 	$(CC) -o $@ $+ $(LDFLAGS)
 
 libx264gtk.a: muxers.o libx264.a

--- x264-0.svn20070613/gtk/Makefile 2007-06-13 15:52:11.000000000 +0200
+++ x264-0.svn20070613/gtk/Makefile 2007-06-13 15:52:41.000000000 +0200
@@ -46,14 +46,14 @@
 
 X264GTK_PC = x264gtk.pc
 
-EXTERNAL_DEPS= ../muxers.o ../matroska.o ../libx264.a
+EXTERNAL_DEPS= ../muxers.o ../matroska.o
 
 
 all: $(ENCODE_BIN) $(TEST_BIN) $(MO_FILES) $(X264GTK_PC)
 
 # Already provides iconv/intl
 CPPFLAGS = -g `pkg-config --cflags gtk+-2.0 gthread-2.0` -I.. -DX264_DATA_DIR=\"${datadir}\"
-LDFLAGS += `pkg-config --libs gtk+-2.0 gthread-2.0` ../libx264.a
+LDFLAGS += `pkg-config --libs gtk+-2.0 gthread-2.0` -L.. -lx264
 
 
 # gettext rules
